### ISSUE TITLE
training models?

### ISSUE THREAD

**Issue by Sugarbank** [yusuketomoto/chainer-fast-neuralstyle#49]
![screen shot 2016-08-27 at 2 22 13 pm](https://cloud.githubusercontent.com/assets/10692268/18025103/e0a4e930-6c61-11e6-9821-845240accd49.png)

Hi, I've tried to train a new model but writes out multiple model files and also .state extension?
eagle sketch was the name of the style image and i trained it with a folder of various images.

any ideas?Thanks heaps.
![output](https://cloud.githubusercontent.com/assets/10692268/18025116/46600836-6c62-11e6-9cc3-864f94b17f4f.jpg)
when I process an image with any of these various.model files it looks like this.

**Comment by Sugarbank** [comment]
also here's my terminal

Christians-iMac:chainer-fast-neuralstyle-master zSUGARBANK$ python train.py -s /Volumes/LaCie/DOWNLOADS/eagle_sketch_by_circle00-d5uoxn4.png -d /Volumes/LaCie/DOWNLOADS/101_ObjectCategories -g -1
num traning images: 29
29 iterations, 2 epochs
epoch 0
(epoch 0) batch 0/29... training loss is...214263808.0
(epoch 0) batch 1/29... training loss is...210467088.0
(epoch 0) batch 2/29... training loss is...66296296.0
(epoch 0) batch 3/29... training loss is...19157852.0
(epoch 0) batch 4/29... training loss is...11985358.0
(epoch 0) batch 5/29... training loss is...11278733.0
(epoch 0) batch 6/29... training loss is...6982236.0
(epoch 0) batch 7/29... training loss is...8931690.0
(epoch 0) batch 8/29... training loss is...8320654.0
(epoch 0) batch 9/29... training loss is...8466308.0
(epoch 0) batch 10/29... training loss is...7682370.5
(epoch 0) batch 11/29... training loss is...6990806.5
(epoch 0) batch 12/29... training loss is...7355416.5
(epoch 0) batch 13/29... training loss is...6850945.5
(epoch 0) batch 14/29... training loss is...7621940.5
(epoch 0) batch 15/29... training loss is...6936177.0
(epoch 0) batch 16/29... training loss is...6777280.5
(epoch 0) batch 17/29... training loss is...6808056.5
(epoch 0) batch 18/29... training loss is...6797888.0
(epoch 0) batch 19/29... training loss is...6722624.0
(epoch 0) batch 20/29... training loss is...6578196.0
(epoch 0) batch 21/29... training loss is...6753231.0
(epoch 0) batch 22/29... training loss is...6510967.5
(epoch 0) batch 23/29... training loss is...6546028.5
(epoch 0) batch 24/29... training loss is...6157487.0
(epoch 0) batch 25/29... training loss is...6392862.5
(epoch 0) batch 26/29... training loss is...6195738.5
(epoch 0) batch 27/29... training loss is...6367346.5
(epoch 0) batch 28/29... training loss is...6199887.5
save "style.model"
epoch 1
(epoch 1) batch 0/29... training loss is...6227217.5
(epoch 1) batch 1/29... training loss is...6481475.5
(epoch 1) batch 2/29... training loss is...6153410.0
(epoch 1) batch 3/29... training loss is...6233668.5
(epoch 1) batch 4/29... training loss is...6321361.0
(epoch 1) batch 5/29... training loss is...6113050.0
(epoch 1) batch 6/29... training loss is...6322238.5
(epoch 1) batch 7/29... training loss is...6375432.5
(epoch 1) batch 8/29... training loss is...6342952.5
(epoch 1) batch 9/29... training loss is...6558215.0
(epoch 1) batch 10/29... training loss is...6081562.5
(epoch 1) batch 11/29... training loss is...6099289.5
(epoch 1) batch 12/29... training loss is...5926570.5
(epoch 1) batch 13/29... training loss is...5780968.0
(epoch 1) batch 14/29... training loss is...6671057.5
(epoch 1) batch 15/29... training loss is...5961956.5
(epoch 1) batch 16/29... training loss is...5816135.0
(epoch 1) batch 17/29... training loss is...5929303.5
(epoch 1) batch 18/29... training loss is...5987084.0
(epoch 1) batch 19/29... training loss is...5909796.0
(epoch 1) batch 20/29... training loss is...5760313.5
(epoch 1) batch 21/29... training loss is...5950045.0
(epoch 1) batch 22/29... training loss is...5718470.5
(epoch 1) batch 23/29... training loss is...5933241.5
(epoch 1) batch 24/29... training loss is...5629819.0
(epoch 1) batch 25/29... training loss is...5786327.5
(epoch 1) batch 26/29... training loss is...5510533.5
(epoch 1) batch 27/29... training loss is...5703062.0
(epoch 1) batch 28/29... training loss is...5626608.5
save "style.model"

Can't find style.model just a whole bunch of different files named the same as the style image
![screen shot 2016-08-27 at 2 22 13 pm](https://cloud.githubusercontent.com/assets/10692268/18025127/a2756242-6c62-11e6-89fe-22b502987a61.png)

**Comment by DylanAlloy** [comment]
if you're only training 29 batches, your results will not look very good.

**Comment by neelkadia-zz** [comment]
@Sugarbank Try to train it with cocodataset
